apps/teams-test-app/e2e-test-data/pages.json

```
@@ -22,9 +22,25 @@
       "type": "callResponse",
       "boxSelector": "#box_returnFocus",
       "checkboxState": true,
-      "expectedAlertValue": "returnFocus called with true",
+      "version": ">2.27.0",
+      "hostSdkVersion": {
+        "web": ">=4.2.0"
```

```suggestion
        "web": ">4.1.0"
```
4.1.0 is current web host sdk version.